doc/api/child_process.markdown

```diff
@@ -755,7 +784,7 @@ grep.stdin.end();
 * `message` {Object}
 * `sendHandle` {Handle object}
 * `callback` {Function}
-* Return: Boolean
+* Return: Boolean | Promise
```

Comment:
Is it common to node APIs or `process.send()` is an outlier? I don't remember ever using that return-and-callback-too pattern.
Let me explain my concerns: "return a promise if no callback was provided" seems to be the least annoying way to support async/await. The other option is having a parallel API. Is `process.send()` the only thing that holds it back?